tests/test_framework/rpc/floresta.py

```diff
@@ -0,0 +1,146 @@
+"""
+floresta_rpc.py
+
+A test framework for testing JsonRPC calls to a floresta node.
+"""
+
+import re
+
+from test_framework.rpc.base import BaseRPC
+
+REGTEST_RPC_SERVER = {
+    "host": "127.0.0.1",
+    "ports": {"rpc": 18442, "server": 50001},
```

Comment:
b4feca68e76b9b33334ca5bf7ecae7e3eb2188c2

Electrum server. Server is too vague